explorer/rest/tests/facade/test_NemRestFacade.py

```diff
@@ -1,22 +1,18 @@
-import unittest
-
-import testing.postgresql
-
 from rest.facade.NemRestFacade import NemRestFacade
 
 from ..db.test_NemDatabase import BlockQueryParams
-from ..test.DatabaseTestUtils import DatabaseConfig, initialize_database
+from ..test.DatabaseTestUtils import DatabaseTestBase
 
 # region test data
 
 EXPECTED_BLOCK_1 = {
```

Comment:
how is this different from BLOCK_VIEWS[0] ?

Reply:
BLOCK_VIEWS[0] is the BlockView class.

EXPECTED_BLOCK_1 is in  `dict` format.

I can do something like this 
`EXPECTED_BLOCK_1 = BLOCK_VIEWS[0] .to_dict()`

Reply:
`EXPECTED_BLOCK_1 = BLOCK_VIEWS[0] .to_dict()` probably is cleaner imo since they need to be consistent anyway